addon/index.js

```diff
@@ -157,6 +158,15 @@ function initServerEnvironmentPreference() {
   });
 }
 
+function openOnboardingTab() {
+  // TODO: Settings is populated async, so we need a global afterSettingsReady
+  // promise that this function can wait on.
+  tabs.open({
+    url: SERVER_ENVIRONMENTS.production.BASE_URL + '/onboarding',
```

Comment:
Hmm, a little leary of hardcoding production here. But, the only idea I have to fix it is already in the TODO so...